helm/ingress-controller/.helmignore

```diff
@@ -0,0 +1,25 @@
+# Source: https://github.com/helm/helm/blob/main/pkg/repo/repotest/testdata/examplechart/.helmignore
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
```

Comment:
thoughts on converting this to an allow list instead of block list?
https://github.com/ngrok/ngrok-ingress-controller/blob/main/.dockerignore

Reply:
I'd probably err on the side of having extra tests files getting included in a release and us fixing it in a dot release than accidentally publishing something that doesn't include a new support files folder. We could move to an allow list, but we'd need some way to test helm packaging locally and make sure we are installing from the package first IMO so we catch not allowing files through that need to be bundled in the chart